Components/Model/Cart/LineItem.php

```diff
@@ -34,7 +34,7 @@
  * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
```

Comment:
Extend https://github.com/Nosto/nosto-php-sdk/blob/develop/src/Object/Cart/LineItem.php instead

Reply:
Maybe we need in order to use the loadData method. I'm not aware if we can load those attributes with the SDK.